### DISCUSSION TITLE
Allow CTRL+Z in "save as" window.

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#364] [Ideas]
Sometimes when navigating to the target folder I want to save a file in, I click on a file instead of a folder, and the file name in the file name field changes to the file I clicked on.

I can't get the original file name back so I have to type it again.

Please allow <kbd>CTRL</kbd>+<kbd>Z</kbd> in the "save as" window.

**Comment by JosephMcc** [comment]
Linux Mint does not control these dialogs. They are provided by whatever toolkit is used to create the application. Gtk or Qt in most cases. The request needs to go to those projects.
